### PR TITLE
[BOT] Farabi/bot-1338/add-permanent-banner-to-bbot

### PR DESCRIPTION
**Pull request by farabi-deriv**
## Changes:

- Created a permanent banner to move to deriv for binary bot)

### Screenshots:

Screenshot 2024-03-25 at 3 44 07 PM

### CODE REVIEW

File: src/components/Banner/redirect-to-dbot.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import GTM from '@utilities/integrations/gtm';
+import { getActiveLoginId } from '@storage';
+
+export const visitDerivBot = () => {
+    const user_id = getActiveLoginId();
+    GTM.pushDataLayer({ event: 'bbot_moved_to_deriv_bot', user_id: user_id ?? null });
+    window.open('https://app.deriv.com/bot/?redirect_from_bbot=1', '_self', 'noopener');
```
Comment (vinu-deriv) [inline src/components/Banner/redirect-to-dbot.js:7]: if in staging, they should be redirected to staging. can we check it from process.env.NODE_ENV or from the url itself using a regex ?

Comment (farabi-deriv) [reply src/components/Banner/redirect-to-dbot.js:11]: Sure @vinu-deriv. Added staging redirection with process.env.NODE_ENV

**Review by rupato-deriv — APPROVED** [review]

**Review by shafin-deriv — APPROVED** [review]

File: src/components/Banner/redirect-to-dbot.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { getActiveLoginId } from '@storage';
+
+export const visitDerivBot = () => {
+    const user_id = getActiveLoginId();
+    GTM.pushDataLayer({ event: 'bbot_moved_to_deriv_bot', user_id: user_id ?? null });
+    const isStaging = process.env.NODE_ENV === 'staging';
+    if (isStaging) {
+        window.open('https://staging-app.deriv.com/bot/?redirect_from_bbot=1', '_self', 'noopener');
+    }
+    window.open('https://app.deriv.com/bot/?redirect_from_bbot=1', '_self', 'noopener');
```
Comment (sandeep-deriv) [inline src/components/Banner/redirect-to-dbot.js:11]: Can we use `getRelatedDeriveOrigin()` to get the origin and then add `redirect_from_bbot=1` ?
```suggestion
  const { origin } = getRelatedDeriveOrigin();
  const url = origin + '?redirect_from_bbot=1';
  window.open(url);
```

**Review by sandeep-deriv — APPROVED** [review]

File: src/components/Banner/redirect-to-dbot.js (hunk 2 above)
Comment (farabi-deriv) [reply src/components/Banner/redirect-to-dbot.js]: Sure. @sandeep-deriv

**Review by vinu-deriv — APPROVED** [review]

**Pull request merged**